review.html

```diff
@@ -0,0 +1,81 @@
+
+    <iframe class="iframe" width="760" height="490" src="https://www.youtube.com/embed/gXLjWRteuWI" 
+    title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; 
+    encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+    <div class="video-details">
+        <a href="#" class="video-title">Learn HTML and CSS Crash Course</a>
+    </div>
+    <div class="discription">
+        <span>12K views</span>
+        <h3 class="view">&#x2022;</h3>
+        <span class="span">3 weeks ago</span>
+    </div>
+
+    <div class="categories">
+        <section class="category-section">
+            <button class="category active">All</button>
+            <button class="category">Category 1</button>
+            <button class="category">Category 2</button>
+            <button class="category">Category 3</button>
+            <button class="category">Category 4</button>
+            <button class="category">Category 5</button>
+            <button class="category">Category 6</button>
+        </section>
+    </div>
+
+    <div class="sidebar">
+        <iframe class="playlist" width="160" height="115" src="https://www.youtube.com/embed/videoseries?list=PLuN9TPzTRAmvwVwxSghtoc9Ems0BkICZl" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+        <iframe class="playlist1" width="160" height="115" src="https://www.youtube.com/embed/iZhV0bILFb0" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+        <iframe class="playlist2" width="160" height="115" src="https://www.youtube.com/embed/Y7jBIH9Ao8U" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+        <iframe class="playlist3" width="160" height="115" src="https://www.youtube.com/embed/27JtRAI3QO8" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+        <iframe class="playlist4" width="160" height="115" src="https://www.youtube.com/embed/HA7hWH2GLjI" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+        <iframe class="playlist5" width="160" height="115" src="https://www.youtube.com/embed/TNV0_7QRDwY" 
+        title="YouTube video player" frameborder="0" 
+        allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" 
+        allowfullscreen></iframe>
+
+    </div>
```

Comment:
Good work for creating the sidebar like that! 🥇 In the meanwhile I would suggest following the best practices of Html and CSS for giving proper indentations. 👍